pkg/psmdb/pmm.go

```diff
@@ -252,6 +252,10 @@ func pmmAgentEnvs(spec api.PMMSpec, secret *corev1.Secret, customLogin bool, cus
 			Name:  "PMM_ADMIN_CUSTOM_PARAMS",
 			Value: customAdminParams,
 		},
+		{
+			Name:  "PMM_AGENT_PATHS_TEMPDIR",
+			Value: "/tmp/pmm-agent",
```

Comment:
maybe we need to use /data/db/pmm-agent

Reply:
what's the purpose of this path?

Reply:
ok, should we use `/data/db/pmm2/tmp` in this case?